app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -211,8 +211,9 @@ class LoginFragment : Fragment() {
     }
 
     private fun showSnackbar() {
-        safeArgs.snackbarMessage?.let { textSnackbar ->
-            Snackbar.make(rootView.loginCoordinatorLayout, textSnackbar, Snackbar.LENGTH_SHORT).show()
+        val snackMessage = safeArgs.snackbarMessage
+        if (snackMessage != "not-set") {
+            Snackbar.make(rootView.loginCoordinatorLayout, snackMessage, Snackbar.LENGTH_SHORT).show()
```

Comment:
Just check snackMessage is `nullOrEmpty()`, Then only show the Snack bar. No need to compare strings. Also do not remove null safety.

Reply:
actually if `snackmessage is set null` , `safeArgs itself become null`. Its Nav AAC issue,  
can't set @null to default value

Reply:
Use empty string then, for default value.

Reply:
>haroldAdmin:  You can leave it as `""`. It shows a red line when you do this, but the project builds and runs just fine.
> thelimitbreaker: That's not a good practice. Are you sure?

Thats the issue. Further I think , setting a default value like 'not-set' (specifically in this case only) make it readable and understandable. Also No red lines, no bad practices

Reply:
What is the logic for red lines?

Reply:
I think the lint check just warns that there's no value set here. It's a warning, not a compilation error.